test/vstest.console.UnitTests/Internal/ConsoleLoggerTests.cs

```diff
@@ -1046,6 +1046,51 @@ public void AttachmentInformationShouldBeWrittenToConsoleIfAttachmentsArePresent
             this.mockOutput.Verify(o => o.WriteLine(string.Format(CultureInfo.CurrentCulture, CommandLineResources.AttachmentOutputFormat, uriDataAttachment1.Uri.LocalPath), OutputLevel.Information), Times.Once());
         }
 
+        [TestMethod]
+        public void ResultscountInhierarchicalOrder()
+        {
+            var loggerEvents = new InternalTestLoggerEvents(TestSessionMessageLogger.Instance);
+            loggerEvents.EnableEvents();
+            var parameters = new Dictionary<string, string>();
+            parameters.Add("verbosity", "normal");
+            this.consoleLogger.Initialize(loggerEvents, parameters);
+
+            TestCase testCase1 = CreateTestCase("TestCase1");
+            TestCase testCase2 = CreateTestCase("TestCase2");
```

Comment:
>CreateTestCase [](start = 33, length = 14)

Does any test cover scenario for non heirarchichal tests as well?

Reply:
yes,
TestRunCompleteHandlerShouldWriteToConsoleIfTestsFail
TestRunCompleteHandlerShouldWriteToConsoleIfTestsPass
These cover  non heirarchichal tests  scenario